rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NSCreationWindow.java

```diff
@@ -208,8 +206,9 @@ public void onFormItemClick(FormItemIconClickEvent event) {
                             .substring(inf.getPluginName().lastIndexOf('.') + 1);
                     values.put(inf.getPluginName(), shortName);
 
-                    ArrayList<FormItem> forms = new ArrayList<FormItem>();
-                    for (Field f : inf.getConfigurableFields()) {
+                    List<Field> configurableFields = inf.getConfigurableFields();
```

Comment:
This logic seems the same of the previous one. Could we extract some common method here?